### PR TITLE
review improvements

### PR DESCRIPTION
**Pull request by nk-coding**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,81 @@
...
\ No newline at end of file
+For the full documentation, have a look at our [website](https://graphglue.github.io)
+
+## Installation
+
+### Full Installation
+
+#### Gradle
+```kts
+implementation("io.github.graphglue", "graphglue", "4.1.2")
```
Comment (chriku) [inline README.md:12]: do not give version

File: README.md [hunk 2]
```diff
@@ -1,4 +1,81 @@
...
+
+Build the project:
+```sh
+./gradlew clean build
+```
+
+Deploy to the local maven repository
+```sh
+./gradlew publishToMavenLocal
+```
```
Comment (chriku) [inline README.md:77]: how to use then?

File: website/docs/docs/authorization.mdx [hunk 3]
```diff
@@ -23,14 +23,14 @@ Declaraing a permission looks like this:
 @DomainNode
 @Authorization(
     name = "READ",
-    allow = [Rule("leafAllowedBean")],
-    allowFromRelated = ["tree"],
-    disallow = [Rule("leafDisallowedBean")]
+    allow = [Rule("actorAllowedBean")],
```
Comment (chriku) [inline website/docs/docs/authorization.mdx:26]: full graph image

File: website/docs/docs/modeling.mdx [hunk 4]
```diff
@@ -15,35 +15,37 @@ Almost all of their features still work when using this library. For more inform
 
 ## Overview
 
-Consider the following example of a `Node`:
+Consider a simple domain model, consisting of movies, actors and directors:
```
Comment (chriku) [inline website/docs/docs/modeling.mdx:18]: image!!!

File: website/docs/docs/modeling.mdx [hunk 5]
```diff
@@ -15,35 +15,37 @@ Almost all of their features still work when using this library. For more inform
...
     
-    fun ageInDays() = age * 365                     // (8)
+    fun actorsCount() = actors.size                   // (8)
 }
 ```
 
 1. Necessary annotation for all `Node` classes. If the `name` parameter is provided, a connection-like query with the name `name` is exposed.
 1. Specifies the `READ` permission for this Node, for more info, see [Authorization](./authorization)
-1. Property which is exposed in the GraphQL schema. As it is annotated with both `@OrderProperty` and `@FilterProperty`, connections of type `Tree` can both be filtered and ordered by this property.
+1. Property which is exposed in the GraphQL schema. As it is annotated with both `@OrderProperty` and `@FilterProperty`, connections of type `Movie` can both be filtered and ordered by this property.
```
Comment (chriku) [inline website/docs/docs/modeling.mdx:44]: ordered where?

File: website/docs/docs/modeling.mdx [hunk 6]
```diff
@@ -15,35 +15,37 @@ Almost all of their features still work when using this library. For more inform
...
 1. Internal property NOT exposed in the GraphQL schema.
 1. Each class needs to extend `Node` in order to use any GraphGlue specific features.
-1. An outgoing relation of type `Root`.
-1. An outgoing relation of type `Set<Leaf>`. In the GraphQL schema, it is exposed as `LeafConnection`
+1. An outgoing relation of type `Director`, each movie has exactly one director.
```
Comment (chriku) [inline website/docs/docs/modeling.mdx:47]: wtf are outgoing?

File: website/docs/docs/modeling.mdx [hunk 7]
```diff
@@ -89,6 +91,11 @@ Like properties, functions can be used to expose field in the GraphQL schema.
 
 ## Relationships
 
+GraphGlue supports two types of relationships: native GraphGlue relationships, and Spring Data Neo4j relationships.
+Both support many-to-many, many-to-one and one-to-one relationships by combining one- and many-sides of relationships.
+Both bidirectional (defining both ends) and unidirectional (defining only one end) relationships are supported.
```
Comment (chriku) [inline website/docs/docs/modeling.mdx:96]: how?

**Review by chriku — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nk-coding) [reply README.md:12]: without a version basically useless, as cannot be copy pasted

File: README.md (hunk 2 above)
Comment (nk-coding) [reply README.md:77]: it is not our responsibility to explain the user how to use maven local  
additionally, for maven it works out of the box

File: website/docs/docs/authorization.mdx (hunk 3 above)
Comment (nk-coding) [reply website/docs/docs/authorization.mdx:26]: uml diagrams are uncommen in the GraphQL context, as GraphQL provides its own non-graphical modeling language

File: website/docs/docs/modeling.mdx (hunk 4 above)
Comment (nk-coding) [reply website/docs/docs/modeling.mdx:18]: see other comment

File: website/docs/docs/modeling.mdx (hunk 5 above)
Comment (nk-coding) [reply website/docs/docs/modeling.mdx:44]: connections can be ordered  
the where is explained by the connections section

File: website/docs/docs/modeling.mdx (hunk 6 above)
Comment (nk-coding) [reply website/docs/docs/modeling.mdx:47]: outgoing is a term to describe the direction of a relation

File: website/docs/docs/modeling.mdx (hunk 7 above)
Comment (nk-coding) [reply website/docs/docs/modeling.mdx:96]: please elaborate

File: website/docs/docs/modeling.mdx (hunk 6 above)
Comment (nk-coding) [reply website/docs/docs/modeling.mdx:47]: 89576acb59e4447532d4f32b0d5a638de47a0f70

**Review by chriku — APPROVED** [review]

**Pull request merged**